x-pack/plugins/security_solution/public/cases/components/all_cases/compact_columns.tsx

```diff
@@ -0,0 +1,284 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useMemo, useCallback, useRef, useState } from 'react';
+import classnames from 'classnames';
+import {
+  EuiAvatar,
+  EuiBadgeGroup,
+  EuiBadge,
+  EuiBasicTable,
+  EuiEmptyPrompt,
+  EuiTableActionsColumnType,
+  EuiTableComputedColumnType,
+  EuiTableFieldDataColumnType,
+  EuiTableSortingType,
+} from '@elastic/eui';
+import { EuiTableSelectionType } from '@elastic/eui/src/components/basic_table/table_types';
+import styled from 'styled-components';
+import { CaseStatuses, CaseType } from '../../../../../cases/common/api';
+import { getEmptyTagValue } from '../../../common/components/empty_value';
+import { Case, DeleteCase, SubCase, SortFieldCase } from '../../containers/types';
+import { getCreateCaseUrl, useFormatUrl } from '../../../common/components/link_to';
+import { useGetCases } from '../../containers/use_get_cases';
+import { useGetCasesStatus } from '../../containers/use_get_cases_status';
+import { EuiBasicTableOnChange } from '../../../detections/pages/detection_engine/rules/types';
+import { CaseDetailsLink, LinkButton } from '../../../common/components/links';
+import { SecurityPageName } from '../../../app/types';
+import { useKibana } from '../../../common/lib/kibana';
+import { APP_ID } from '../../../../common/constants';
+import { SELECTABLE_MESSAGE_COLLECTIONS } from '../../translations';
+import * as i18n from './translations';
+import { Status } from '../status';
+import { getSubCasesStatusCountsBadges, isSubCase } from './helpers';
+
+export type CasesColumns =
+  | EuiTableFieldDataColumnType<Case>
+  | EuiTableComputedColumnType<Case>
+  | EuiTableActionsColumnType<Case>;
+
+const MediumShadeText = styled.p`
+  color: ${({ theme }) => theme.eui.euiColorMediumShade};
+`;
+
+const Spacer = styled.span`
+  margin-left: ${({ theme }) => theme.eui.paddingSizes.s};
+`;
+
+const TagWrapper = styled(EuiBadgeGroup)`
+  width: 100%;
+`;
+
+export const getCompactCasesColumns = (): CasesColumns[] => {
```

Comment:
is this file used?  my guess is that it's left over after this was moved out for RAC